llama_stack/providers/inline/vector_io/sqlite_vec/sqlite_vec.py

```diff
@@ -63,12 +65,19 @@ async def initialize(self) -> None:
             CREATE VIRTUAL TABLE IF NOT EXISTS {self.vector_table}
             USING vec0(embedding FLOAT[{self.dimension}], id TEXT);
         """)
+        # FTS5 table (for keyword search) - creating both the tables by default. Will use the relevant one based on
```

Comment:
I would only create the table if its enabled.

Reply:
We create it by default during initialisation, so that while querying the user has the option to chose if they want to do a vector/keyword/hybrid search. If we don't want to create it by default - then while initialising the DB the user would have to particularly chose the type of query search and just stick to it. The former provides flexibility. Wdyt?

Reply:
Yeah, that's a good point! Agree. 👍

Reply:
That being said, I do see the other side of it. Where the data is huge and we spend memory creating vectors or keywords unnecessarily when the user clearly knows what they want.

Reply:
Will leave it to default initialisation for now, then. We can provide more knobs for user to customise. It shouldn't be an immediate issue, given sqlite is not usually used in production.